README.md

```diff
@@ -10,7 +10,7 @@ For details about this tool, please consult one of the following sources:
 Contributions are welcome, please look over [CONTRIBUTING.md](CONTRIBUTING.md) for more information on how to get started.
 
 ## Contact
-
+Line added by Adam :)
```

Comment:
So far so good! 
Now I ask you to update this line. After updating it should look like this:
```
# This simple line was added by Adam :)
```
It should be the same commit and you should update it from the command-line.
Hint:
take a look at the `--amend` parameter :)

Reply:
Sorry, I missed your review and I have already deleted this fork (I did it, when I was creating fork from oamg).